controller/api/api.go

```diff
@@ -612,10 +612,14 @@ func (a *Api) scaleContainer(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := a.manager.ScaleContainer(containerId, numInstances); err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+	result := a.manager.ScaleContainer(containerId, numInstances)
+	json, _ := json.Marshal(result)
```

Comment:
Yeah, I was debating what to do with this error too.  I figured that the result struct that is returned is so simple that I couldn't think of any error scenarios that might occur when marshalling it.  So I felt adding in a check for the error would just be adding unnecessary clutter.. but yeah, I might be wrong -- what do you think? :)